lib/reporting/drop_off_report.rb

```diff
@@ -388,6 +388,11 @@ def query
             name
           , properties.user_id AS user_id
           , coalesce(properties.event_properties.success, 0) AS success
+          , coalesce(properties.event_properties.fraud_review_pending, 0) AS fraud_review_pending
+          , coalesce(properties.event_properties.gpo_verification_pending, 0) AS gpo_verification_pending
+          , coalesce(properties.event_properties.in_person_verification_pending, 0) AS in_person_verification_pending
+          , ispresent(properties.event_properties.deactivation_reason) AS has_other_deactivation_reason
+          , !fraud_review_pending and !gpo_verification_pending and !in_person_verification_pending and !has_other_deactivation_reason AS identity_verified
```

Comment:
can you update the specs with examples of a few of these fields coming back in the response?